frontend/interfaces/software.ts

```diff
@@ -90,7 +90,7 @@ export interface ISoftwareTitle {
   id: number;
   name: string;
   versions_count: number;
-  source: string;
+  source: string; // ipados_apps | ios_apps | apps | ...
```

Comment:
nit: should the comment in line 93 match the comment in line 106

Reply:
sure!

Reply:
I'll put that in the follow-up PR